mathics/core/systemsymbols.py

```diff
@@ -70,7 +70,6 @@
 SymbolFailed = Symbol("System`$Failed")
 SymbolFloor = Symbol("System`Floor")
 SymbolFormat = Symbol("System`Format")
-SymbolFullForm = Symbol("System`FullForm")
```

Comment:
Why was this removed here? Removing it makes the current mathicsscript break because this is currently where it expects it.

I do see it also defined in `mathics/core/symbols.py` but it is not _required_ to be there. And current thinking is that we should be explicit about adding the context "System" if that is what we mean rather because it is more clear, rather than assuming the context is "System" when the symbol is defined as it is in `mathics/core/symbols.py`

Reply:
Actually, these symbols appear in `mathics.core.symbols` because the formatting was inside the module, and required them. Now probably can be removed from there and moved to mathics.core.systemsymbols.

Reply:
Please move. Thanks.

Reply:
Done here: [2bca2aa](https://github.com/Mathics3/mathics-core/pull/482/commits/2bca2aaac08de5b20f510ded69bcb8bb733e29dc)